README.md

```diff
@@ -1,24 +1,144 @@
-application up and running.
+### Description
 
-Things you may want to cover:
+Nick and Kyle's [Rales Engine](http://backend.turing.io/module3/projects/rails_engine) project for Turing. In this project we import business data and host an API with endpoints to return records, relationships between models, and business intelligence as JSON. Checkout the schema file `schema.png` in the public directory to see the models and their relationshps.
 
-* Ruby version
+Tech stack: Ruby, Rails, PostgreSQL, RSpec
 
-* System dependencies
+### Dependencies 
 
-* Configuration
+This project uses Ruby version 2.3.3 with a PostgreSQL database. The dataset is from the [sales engine](https://github.com/turingschool-examples/sales_engine/tree/master/data) repository.
 
-* Database creation
+### Getting Started
 
-* Database initialization
+Follow these steps in your terminal to clone the project on to your local machine and import the data.
 
-* How to run the test suite
+  1. `cd` into the directory where you want the project in the terminal.
+  1. Run `git clone https://github.com/ski-climb/rales_engine.git`
+  1. `cd rales_engine` 
+  1. `bundle` to install the gems you need
+  1. `rake db:create` to create your PostgreSQL database
+  1. `rake import:all` to import the data
 
-* Services (job queues, cache servers, search engines, etc.)
+### Test suite
 
-* Deployment instructions
+Follow these steps in your terminal to run our test suite and check out the coverage.
 
-* ...
+  1. `rspec` to run the test suite
+  1. `open coverage/index.html` to view the test coverage in your browser
+
+### Hosting the site locally.
+
+  1. From within the `rales_engine` directory run `rails server` to start the server locally.
+  1. In your browser visit any provided endpoint to view the response (example: `http://localhost:3000/api/v1/merchants`)
+
+### Running the Spec Harness
+
+This project came with a spec harness. The spec harness checks the project's API endpoints by hitting urls on the local server. Follow these steps in your terminal to run the project agains the spec harness.
+
+  1. From within the `rales_engine` directory run `rails server` to start the server locally.
+  2. Open a new tab within your terminal. Be sure it is a new tab and the server is running in your first terminal tab.
+  3. In your new terminal tab, run `git clone https://github.com/turingschool/rales_engine_spec_harness.git` 
+  4. In the same terminal tab, run `cd rales_engine_spec_harness`
+  5. Still in the same terminal tab, run `rake` to run the spec harness tests.
+
+### Endpoints Provided
+
+The models are:
+  * merchants
+  * customers
+  * items
+  * invoices
+  * invoice_items
+  * transactions
+
+#### Single Finders
+
+For any of the models visit :`GET /api/v1/models/find?parameters`
+
+Parameters should that can be passed in are name, id, created_at, or updated_at.
+
+For example: `GET /api/v1/merchants/find?name=Schroeder-Jerde`
+
```

Comment:
What would you think about adding one more example here to help show what should and should not change as the params are updated.  I think this would be a one time only example (i.e. not necessary anywhere else but here).  We can show the user how to visit the endpoint using the browser:
Another example: `http://localhost:3000/api/v1/merchants/find?id=3`